cedar-policy-validator/src/human_schema/to_json_schema.rs

```diff
@@ -407,11 +404,7 @@ impl<'a> ConversionContext<'a> {
 }
 
 /// Wrap [`ExactlyOneError`] for the purpose of converting PRDecls
-fn convert_pr_error(
-    _e: ExactlyOneError<std::vec::IntoIter<Vec<SmolStr>>>,
-    kind: PR,
-    loc: Loc,
-) -> ToJsonSchemaErrors {
+fn convert_pr_error(kind: PR, loc: Loc) -> ToJsonSchemaErrors {
```

Comment:
This doc comment can probably be updated

Reply:
This function is no longer needed.